docs/features/software-templates/installation.md

```diff
@@ -0,0 +1,185 @@
+# Installing in your Backstage App
+
+The scaffolder plugin comes in two packages, `@backstage/plugin-scaffolder` and
+`@backstage/plugin-scaffolder-backend`. Each has their own installation steps,
+outlined below.
+
+The Scaffolder plugin also depends on the Software Catalog. Instructions for how
+to set that up can be found [here](./TODO.md).
+
+## Installing @backstage/plugin-scaffolder
+
+`The scaffolder frontend plugin should be installed in your`app`package, which is created as a part of`@backstage/create-app`.
+To install the package, run:
+
+```bash
+cd packages/app
+yarn add @backstage/plugin-scaffolder
+```
+
+Make sure the version of `@backstage/plugin-scaffolder` matches the version of
+other `@backstage` packages. You can update it in `packages/app/package.json` if
+it doesn't.
+
+### Adding the Plugin to your `packages/app`
+
+Add the following entry to the head of your `packages/app/src/plugins.ts`:
+
+```ts
+export { plugin as ScaffolderPlugin } from '@backstage/plugin-scaffolder';
+```
+
+Add the following to your `packages/app/src/apis.ts`:
+
+```ts
+import { scaffolderApiRef, ScaffolderApi } from '@backstage/plugin-scaffolder';
+
+// Inside the ApiRegistry builder function ...
+
+builder.add(
+  scaffolderApiRef,
+  new ScaffolderApi({
+    apiOrigin: backendUrl,
+    basePath: '/scaffolder/v1',
+  }),
+);
+```
+
+Where `backendUrl` is the `backend.baseUrl` from config, i.e.
+`const backendUrl = config.getString('backend.baseUrl')`.
+
+This is all that is needed for the frontend part of the Scaffolder plugin to
+work!
+
+## Installing @backstage/plugin-scaffolder-backend
+
+The scaffolder backend should be installed in your `backend` package, which is
+created as a part of `@backstage/create-app`. To install the package, run:
```

Comment:
I like that information, but the current template is [fully configured for that.](https://github.com/spotify/backstage/blob/master/packages/create-app/templates/default-app/packages/backend/src/plugins/scaffolder.ts).
Maybe we can add an alert for that, something like: "Your backend is already configured when using our create-app, but If you create a project from scratch, you can do follow that steps".

Reply:
Yeah that's a good point. We wrote these docs as bit of an exercise and template for other installation docs, and simply to make sure we know of all things that are required. Should definitely mention that the backend already comes with this though.